contrib/terraform/vsphere/cloud-init.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# cloud-config
+
+# sets the hostname to 'terraformed'
+hostname: terraformed
+
+ssh_authorized_keys:
+- ${ssh_pub_key}
+
+write_files:
+- path: /etc/netplan/20-internal-network.yaml
+  content: |
+    network:
+      version: 2
+      ethernets:
+        "lo:0":
+          match:
+            name: lo
+          dhcp4: false
+          addresses:
+          - 172.17.0.100/32
+- path: /etc/netplan/10-user-network.yaml
+  content: |
+    network:
+      version: 2
+      ethernets:
+        ens192:
+          dhcp4: false #true to use dhcp
+          addresses:
+          - ${ip}
+          gateway4: ${gw} # Set gw here
+          nameservers:
+            addresses:
+            - ${dns} # Set DNS ip address here
+
+runcmd:
+- netplan apply
```

Comment:
Do these belong in Kubespray? If yes, then please make a visible note in `README.md` to clearly signal that the Terraform script assumes lack of DHCP.